packages/bun-debug-adapter-protocol/src/debugger/adapter.ts

```diff
@@ -215,6 +215,7 @@ export type DebugAdapterEventMap = InspectorEventMap & {
   "Process.requested": [unknown];
   "Process.spawned": [ChildProcess];
   "Process.exited": [number | Error | null, string | null];
+  "Process.success": [undefined | Error , undefined];
```

Comment:
The signature for `Process.exited` did not make sense to me, so this is just a guess. I don't know if I can figure this one out without help either.

Reply:
This change is unecessary/wrong - this object represents events that get emitted in the debug inspector protocol (the websocket events sent when you do `bun --inspect`). We are already emitting Process.exited but we won't be changing to emit Process.success

For Process.exited, it accepts two arguments - an exit code or an Error instance if it failed, and a signal

Currently this is emitted here: https://github.com/oven-sh/bun/blob/be03a537df655cd4ed3f8057d4a71c5efa168d65/packages/bun-debug-adapter-protocol/src/debugger/adapter.ts#L2289